cif_core.dic

```diff
@@ -27546,17 +27667,27 @@ save_
        Changed the purpose of the _diffrn_radiation_wavelength.id data item
        from 'Encode' to 'Key'.
 
+       Reverted _atom_type_scat.versus_stol_list to a text-type to regain
+       compatibility with DDL1 dictionary.
+
        Added _atom_type_scat.exponential_polynomial_coefs, Gaussian_coefs, and
        inv_Mott_Bethe_coefs to allow for an arbitrary number of coefficients in
        the definition of form factors.
+
 ;
          3.3.0                    2023-06-16
 ;
        # Please update the date above and describe the change below until
        # ready for the next release
 
+       Rewrote the _atom_type.radius_contact evaluation method as a definition
+       method.
+
```

Comment:
```suggestion
```

Reply:
Seems like an artifact introduced during the last sync with the `master` branch. Should be removed.